gradle.properties

```diff
@@ -0,0 +1 @@
+version=0.1.0-SNAPSHOT
```

Comment:
Had to add it here, otherwise `project.version` is set to `unspecified` by default and I was reluctant to depend on that arbitrary string in the code